static/js/browser2/out_arc_section.tsx

```diff
@@ -128,36 +118,46 @@ export class OutArcSection extends React.Component<
     });
     Promise.all(propValuesPromises)
       .then((propValuesData) => {
-        const outArcsByPredicate = {};
+        const outArcsByPredicateAndProvenance = {};
         const propertyLabels = [];
         propertyLabels.push("typeOf");
         propValuesData.forEach((valuesData) => {
           if (!valuesData || _.isEmpty(valuesData.values)) {
             return;
           }
           const predicate = valuesData.property;
-          let values = valuesData.values.out;
-          if (
-            PROPERTIES_TO_TRIM.has(predicate) &&
-            values.length > NUM_VALUES_TRIMMED
-          ) {
-            const extra = values.length - NUM_VALUES_TRIMMED;
-            values = _.slice(values, 0, NUM_VALUES_TRIMMED);
-            // TODO (chejennifer): find better way to do this
-            values.push({
-              value: "(... " + extra + " more ...)",
+          const values = valuesData.values.out;
+          for (const value of values) {
+            if (!(predicate in outArcsByPredicateAndProvenance)) {
+              outArcsByPredicateAndProvenance[predicate] = {};
+            }
+            const provId = value.provenanceId;
+            if (!(provId in outArcsByPredicateAndProvenance[predicate])) {
+              outArcsByPredicateAndProvenance[predicate][provId] = [];
+            }
+            let valueText = "";
+            if (value.dcid) {
+              valueText = value.name ? value.name : value.dcid;
+            } else {
+              valueText = value.value;
+            }
+            outArcsByPredicateAndProvenance[predicate][provId].push({
+              dcid: value.dcid,
+              text: valueText,
             });
           }
-          outArcsByPredicate[predicate] = values;
           if (predicate !== "typeOf") {
             propertyLabels.push(predicate);
           }
         });
-        outArcsByPredicate["dcid"] = [{ value: this.props.dcid }];
+        outArcsByPredicateAndProvenance["dcid"] = {};
+        outArcsByPredicateAndProvenance["dcid"]["none"] = [
```

Comment:
No, we don't want the string "none" to appear in the UI because there is no provenance for the dcid of a node. This won't show up in the UI because we only show provenance that have a corresponding URL in a mapping of provenanceId to their URLs. Maybe it'll be more clean to just render another ArcTableRow for dcid in the render method instead of trying to add dcid to the data like this.